src/Application/Model/Ticket.php

```diff
@@ -616,15 +616,6 @@ public static function createMissingEncodingTickets($project) {
 			return $handle->fetch()['create_missing_encoding_tickets'];
 		}
 		
-		public static function createMissingEncodingTicket($metaTicket, $encodingProfile) {
```

Comment:
This method is used when adding an encoding profile to the project in the current behaviour (Controller/Projects.php, around line 91).
Am I missing where the auto create will take place if it is enabled?

Reply:
That's another reason why I want to remove that method. It's so easy to mistake this method as it's plural (Ticket**s**) version right above in the file.

Reply:
Got me, I used only full text search and was confused though I did see both versions of the method.